Remotion/Web/Core/UI/Controls/SingleViewImplementation/Rendering/SingleViewRenderer.cs

```diff
@@ -170,6 +170,7 @@ private void RegisterAdjustViewScript (SingleViewRenderingContext renderingConte
           renderingContext.Control,
           typeof (SingleViewRenderer),
           Guid.NewGuid ().ToString (),
+          // TODO RM-7670 adjust call
```

Comment:
I don't think its worth touching this file just to add this TODO. We already have the todo in the TypeScript file.